backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5773541_MInorImprovementsrelatedToPaySelectionAndPayment.sql

```diff
@@ -0,0 +1,25 @@
+
+
+-- Name: C_DocType AR/AP Invoices and Credit Memos IsSOTrx-Filter
+-- 2025-10-20T09:09:18.711Z
+UPDATE AD_Val_Rule SET Code='C_DocType.DocBaseType IN (''ARI'', ''API'',''ARC'',''APC'') ',Updated=TO_TIMESTAMP('2025-10-20 09:09:18.707000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',UpdatedBy=100 WHERE AD_Val_Rule_ID=124
```

Comment:
which was the previous where clause?
(wondering if is OK the chnge current val rule....)